RobotCode/src/main/java/frc/lib/core/motors/TeamTalonFX.java

```diff
@@ -161,4 +161,42 @@ public void configureWithPidParameters(final PidParameters pidParameters, final
     configPeakOutputReverse(-pidParameters.getKPeakOutput(), 30);
     configAllowableClosedloopError(pidSlotIndex, pidParameters.getErrorTolerance(), 30);
   }
+
+  @Override
+  public void configF(final double F, final int SLOT)
+  {
+    config_kF(SLOT, F, 30);
+  }
+
+  @Override
+  public void configP(final double P, final int SLOT)
+  {
+    config_kP(SLOT, P, 30);
+  }
+
+  @Override
+  public void configI(final double I, final int SLOT)
+  {
+    config_kI(SLOT, I, 30);
+  }
+
+  @Override
+  public void configD(final double D, final int SLOT)
+  {
+    config_kD(SLOT, D, 30);
+  }
+
+  @Override
+  public void configPeakOutput(final double PEAK_OUTPUT)
+  {
+    configPeakOutputForward(PEAK_OUTPUT, 30);
+    configPeakOutputReverse(-PEAK_OUTPUT, 30);
+  }
+
+  @Override
+  public void setClosedLoopErrorLimit(final double ERROR_TOLERANCE, final int SLOT)
+  {
+    configAllowableClosedloopError(SLOT, (int) ERROR_TOLERANCE, 30);
+  }
+
```

Comment:
no onPidPeriodic() body for this class, I don't know if that is something you would want/need to add

Reply:
`onPidPeriodic` has a body in IMotor, so it doesn't have to have a body in implementations.